gui/src/components/DesignStoryComponents/DesignStoryList.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { useState } from 'react';
+import { useDesignContext } from '@/context/Design';
+import { Button } from '@nextui-org/react';
+import styles from '@/components/DesignStoryComponents/story.module.css';
+import { handleStoryStatus } from '@/app/utils';
+import CustomInput from '@/components/CustomInput/CustomInput';
+import imagePath from '@/app/imagePath';
+
+const DesignStoryList: React.FC = () => {
+  const {
+    setOpenCreateStoryModal,
+    setOpenStoryDetailsModal,
+    storyList,
+    setSelectedStory,
+  } = useDesignContext();
+  const [searchTerm, setSearchTerm] = useState('');
+  const handleOpenStoryDetails = (storyDetails) => {
+    setOpenStoryDetailsModal(true);
+    setSelectedStory(storyDetails);
+    localStorage.setItem('storyId', storyDetails.id);
+  };
+  const tagCSS = {
```

Comment:
we have similar const for board as well, so can we have a single const and import it from one of the constant files

Reply:
resolved